README.md

```diff
@@ -19,9 +19,16 @@ file template
 database and the pca-template-stats for looking at templates from the PCA
 Database. REQUIRES OTHER DOCKERS
 
+## New Repositories from a Skeleton ##
+
+Please see our [Project Setup guide](https://github.com/cisagov/development-guide/tree/develop/project_setup)
+for step-by-step instructions on how to start a new repository from
+a skeleton. This will save you time and effort when configuring a
+new repository!
+
```

Comment:
FYI, the block above (`## New Repositories from a Skeleton ## ...`) should be deleted, since it only applies to skeleton repos.

Reply:
The above block was removed as part of https://github.com/cisagov/pca-report-library/pull/3/commits/550dc34b08768dd4ac0d4b38b83a1f6893d4333f